src/js/components/Distribution/__tests__/__snapshots__/Distribution-test.js.snap

```diff
@@ -173,24 +173,48 @@ exports[`Distribution gap renders 1`] = `
   height: 48px;
 }
 
+@media only screen and (max-width:768px) {
+  .c3 {
+    width: 3px;
+  }
+}
+
 @media only screen and (max-width:768px) {
   .c5 {
     height: 3px;
   }
 }
 
+@media only screen and (max-width:768px) {
+  .c7 {
+    width: 6px;
+  }
+}
+
 @media only screen and (max-width:768px) {
   .c8 {
     height: 6px;
   }
 }
 
+@media only screen and (max-width:768px) {
+  .c9 {
+    width: 12px;
+  }
+}
+
 @media only screen and (max-width:768px) {
   .c10 {
     height: 12px;
   }
 }
 
+@media only screen and (max-width:768px) {
+  .c11 {
+    width: 24px;
+  }
+}
+
```

Comment:
Where did all those similar changes come from? are we should this `width` size is backward compatible.

Reply:
Way to question the snapshot changes! The changes actually illustrate a problem that is fixed by this pull request. Previously, the Distribution gap was only responsive in column direction but not row, as you can see here:
![pre-change](https://user-images.githubusercontent.com/11637956/78672816-03a82480-7896-11ea-8e5b-f55698efaed5.png)
But with these changes, both column and row respond down:
![post-change](https://user-images.githubusercontent.com/11637956/78672862-16225e00-7896-11ea-8204-13c4a645359b.png)

Reply:
Great, thanks for the detailed response.